Source/WebCore/platform/graphics/transforms/AffineTransform.h

```diff
@@ -71,6 +72,10 @@ class AffineTransform {
     AffineTransform(const D2D1_MATRIX_3X2_F&);
 #endif
 
+#if PLATFORM(HAIKU)
+	AffineTransform(const BAffineTransform&);
```

Comment:
I would prefer this was not here, or at least marked explicit to avoid accidentally converting between the two (it could be a performance problem). There are already functions to convert between the two types where needed.

Reply:
@pulkomandy so including webkit headers would have BAffineTransform within?

Reply:
No, if you do this:

#include "AffineTransform.h" // get the WebCore header
#include <AffineTransform.h> // get the Haiku header

There are several other cases of similar header clashes in the codebase, which is the reason for introducing LOCAL_INCLUDE_DIRECTORIES in the CMakeLists. The same change can be applied in the CMakeLists for WebKit(2)